pkg/testing/testing_test.go

```diff
@@ -21,4 +21,8 @@ import (
 )
 
 // Hook up gocheck into the "go test" runner.
-func Test(t *testing.T) { TestingT(t) }
+func Test(t *testing.T) {
+	integrationSetup(t)
```

Comment:
We use build tag in integration_test.go, so if we want to use functions defined in that file, we need to add the `integration` tag in build. The reason was to separate out  integration test and unit test execution 
https://github.com/kanisterio/kanister/pull/679/files#diff-3c0035a2cccefd1a523cfcf89d924e62R1

Reply:
Good point.  Didn't notice comment before pushing iteration.  Will think a bit after breakfast.